x-pack/plugins/apm/server/lib/errors/get_error_group.ts

```diff
@@ -84,3 +85,57 @@ export async function getErrorGroup({
     occurrencesCount: resp.hits.total
   };
 }
+
+export async function getErrorCountByTransaction({
+  transaction,
+  setup
+}: {
+  transaction: Transaction;
+  setup: Setup;
+}): Promise<number> {
+  const { start, end, esFilterQuery, client, config } = setup;
+  const filter: ESFilter[] = [
+    { term: { [SERVICE_NAME]: transaction.service.name } },
+    {
+      bool: {
+        should: [
+          {
+            match_phrase: {
+              [TRANSACTION_ID]: transaction.transaction.id
+            }
+          }
+        ],
+        minimum_should_match: 1
+      }
+    },
+    { exists: { field: ERROR_GROUP_ID } },
+    {
+      range: {
+        '@timestamp': {
+          gte: start,
+          lte: end,
+          format: 'epoch_millis'
+        }
+      }
+    }
+  ];
```

Comment:
I don't think you need either of the `should` clauses or the exist clause. This should do the trick:

```json
[
  { term: { [SERVICE_NAME]: transaction.service.name } },
  { term: { [TRANSACTION_ID]: transaction.transaction.id } },
  { term: { [PROCESSOR_EVENT]: 'error' } },
  {
    range: {

      '@timestamp': {
        gte: start,
        lte: end,
        format: 'epoch_millis'
      }
    }
  }
];
```